dev/com.ibm.ws.channel.ssl/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -35,3 +35,5 @@ default.sslRef.desc=The default SSL configuration repertoire. The default value
 ssl.sslSessionTimeout=SSL session timeout
 ssl.sslSessionTimeout.desc=The timeout limit for an SSL session that is established by the SSL Channel. 
 
+ssl.sslSessionCacheSize=SSL session Cache Size
+ssl.sslSessionCacheSize.desc=Limit the number of allowed items in the cache.
```

Comment:
The "Limit" in the desc is a verb, when a noun is expected for a desc. Suggested change: 
`The maximum number of items allowed in the cache.`

If the desc should mean something different, pls tell me.

Reply:
Made the change as suggested.